tests/benchmarks/BenchmarkComparison/Program.fs

```diff
@@ -0,0 +1,11 @@
+﻿
+open System.IO
+open System.Reflection
+open BenchmarkDotNet.Running
+open BenchmarkComparison
+
+[<EntryPoint>]
+let main _ =
+    BenchmarkRunner.Run<TypeCheckingBenchmark1>() |> ignore
```

Comment:
It would be interesting to test it with different JIT and PGO options in future.